app/components/elements/forms/label_component.rb

```diff
@@ -5,18 +5,28 @@ module Forms
     # Component for rendering a form label.
     class LabelComponent < ApplicationComponent
       def initialize(form:, field_name:, label_text: nil, default_label_class: 'form-label', hidden_label: false, # rubocop:disable Metrics/ParameterLists
-                     classes: [])
+                     classes: [], tooltip: nil)
         @form = form
         @label_text = label_text
         @field_name = field_name
         @hidden_label = hidden_label
         @default_label_class = default_label_class
         @classes = classes
+        @tooltip = tooltip
         super()
       end
 
       attr_reader :field_name, :form
 
+      def render_tooltip?
```

Comment:
Opinion: I'd go with `tooltip?` instead of `render_tooltip?`.

Reply:
@justinlittman Sounds good to me. 👍🏻